docs/gates.md

```diff
@@ -1,46 +1,224 @@
-## Gates
+# Gates
 
 A ``Gate`` is an operation that can be applied to a collection of
 qubits (objects with a ``Qid``).  ``Gates`` can be applied
 to qubits by calling their ``on`` method, or, alternatively
 calling the gate on the qubits.  The object created by such calls
 is an ``Operation``.
+
+This can be compared to circuit diagrams often seen in quantum computing
+papers and textbooks.
```

Comment:
It's not clear what "This" is referring to. I would get rid of this sentence.

Reply:
Done.  I explain it in the next section anyway.